packages/flutter_tools/templates/plugin/.idea/workspace.xml.tmpl

```diff
@@ -11,7 +11,7 @@
           </provider>
         </entry>
       </file>
-      <file leaf-file-name="main.dart" pinned="false" current-in-tab="true">
+      <file leaf-file-name="main.dart" pinned="false" current-in-tab="false">
```

Comment:
Does this need to be projevtname.dart in packages?

Reply:
In packages and apps we only open one file. This is for plugins, where we open two files. However, two files get opened only when a project is created by the command-line tool then opened in IntelliJ. If you create the project using the IntelliJ wizard only one file gets opened. That's not a regression but is a bug we should try to fix.

Reply:
Filed https://github.com/flutter/flutter-intellij/issues/1370

Reply:
Got it, thanks